frontend/src/components/data-table/columns.tsx

```diff
@@ -220,30 +132,36 @@ export function generateColumns<T>({
           <DataTableColumnHeaderWithSummary
             header={info.key}
             column={column}
-            formatOptions={formatOptions[dtype as keyof typeof formatOptions]}
             setFormat={(format) =>
               setColumnFormats({ ...columnFormats, [info.key]: format })
             }
             summary={<TableColumnSummary columnId={info.key} />}
           />
         );
       },
-      cell: ({ renderValue, getValue }) => {
+
+      cell: ({ column, renderValue, getValue }) => {
         // Row headers are bold
         if (rowHeadersSet.has(info.key)) {
           return <b>{String(renderValue())}</b>;
         }
 
         const value = getValue();
-        const dtype = fieldTypes?.[info.key]?.[1];
+        const dtype = mapDTypeToDataType(fieldTypes?.[info.key]?.[1]);
```

Comment:
this can be `column.meta.dataType` once its you at it to the meta properties

Reply:
thanks for the review🙂